libapicore/ApiServer.cpp

```diff
@@ -857,18 +857,25 @@ Json::Value ApiConnection::getMinerStatHR()
 
     for (unsigned gpuIndex = 0; gpuIndex < p.minersHashes.size(); gpuIndex++)
     {
-        auto const& minerhashes = p.minersHashes[gpuIndex];
-        auto const& minermonitors = p.minerMonitors[gpuIndex];
-        auto const& miningispaused = p.miningIsPaused[gpuIndex];
+        bool doMonitors = (!p.minerMonitors.empty() && gpuIndex <= (p.minerMonitors.size() - 1));
```

Comment:
Think 
`bool doMonitors = (gpuIndex < p.minerMonitors.size());`
would do the same

Reply:
Therefore I use `<` and not `<=`